base/normalize_unicode.cpp

```diff
@@ -7,23 +7,23 @@
,0x3009,0x3008,0x300b,0x300a,0x3011,0x3010,0x7e,0x2985,0x2986,0x3002,0x640,0x640,0x621,0x630,0x630,0x629,0x629,0x25,0x27,0x3017,0x3016,0x24,0x24,0x22,0x39e,0x39d,0x39c,0x39b,0x39a,0x399,0x398,0x397,0x396,0x395,0x394,0x393,0x392,0x391,0x237,0x131,0x3c0,0x3c1,0x3c6,0x3ba,0x3b8,0x3b5,0x2202,0x3c9,0x3b2,0x3b1,0x2207,0x3a8,0x3a7,0x3a6,0x3a5,0x3a4,0x3a3,0x3a1,0x3a0,0x39f,0x3b6,0x3b4,0x3b3,0x3c8,0x3c7,0x3c5,0x3c4,0x3c3,0x3c2,0x3bf,0x3be,0x3bd,0x3bb,0x3b9,0x3b7,0x3dd,0x3dc,0x11b3,0x11b2,0x11b1,0x11b0,0x1104,0x11ad,0x11ac,0x11aa,0x1101,0x1160,0x110d,0x110a,0x1121,0x1108,0x111a,0x11b5,0x11b4,0x30cc,0x30e8,0x30e2,0x1d1ba,0x1d1b9,0x1d1b9,0x1d158,0x1d158,0x1d157,0x110a5,0x1173,0x1172,0x1171,0x1170,0x116f,0x116d,0x116c,0x116b,0x116a,0x1168,0x1167,0x1166,0x1164,0x1163,0x1162,0x1109b,0x11099,0x25cb,0x25a0,0x2193,0x2192,0x2191,0x2190,0x2502,0x20a9,0xa5,0xa6,0xac,0xa3,0xa2,0x1175,0x292,0x250,0x251,0x1d02,0x259,0x25b,0x25c,0x14b,0x254,0x1d16,0x1d17,0x1d1d,0x26f,0x1025,0x10dc,0x1b05,0x1b07,0x1b09,0x1b0b,0x1b0d,0x1b11,0xc6,0x18e,0x222,0x268,0x269,0x26a,0x1d7b,0x29d,0x26d,0x1d85,0x29f,0x271,0x270,0x272,0x273,0x274,0x275,0x278,0x282,0x283,0x1ab,0x289,0x28a,0x1d1c,0x28b,0x28c,0x290,0x291,0x1d25,0x43d,0x252,0x255,0xf0,0x25f,0x261,0x265,0xe6,0x438,0x443,0x474,0x475,0x416,0x436,0x410,0x430,0x430,0x415,0x435,0x4d8,0x4d9,0x417,0x437,0x418,0x418,0x41e,0x43e,0x4e8,0x4e9,0x413,0x406,0x41a,0x423,0x433,0x456,0x43a,0x91c,0x921,0x922,0x92b,0x92f,0x9a1,0x9a2,0x9af,0xa32,0xa38,0xa16,0xa17,0xa1c,0xa2b,0xb21,0xb22,0xb92,0xe32,0xeb2,0xf0b,0xf42,0xf4c,0xf51,0xf56,0xf5b,0xf40,0x42d,0x44d,0x427,0x447,0x42b,0x44b,0x6c1,0x928,0x930,0x933,0x915,0x916,0x917,0x1b7,0xd8,0xf8,0x266,0x279,0x27b,0x281,0x263,0x295,0x2b9,0x1158,0x1159,0x1184,0x1185,0x1188,0x1191,0x1192,0x1194,0x119e,0x11a1,0x11d3,0x11d7,0x11d9,0x111c,0x11dd,0x11df,0x111d,0x111e,0x1120,0x1122,0x1123,0x1127,0x1129,0x112b,0x112c,0x112d,0x112e,0x112f,0x1132,0x1136,0x1140,0x1147,0x114c,0x11f1,0x11f2,0x1157,0x30f0,0x30f1,0x30fd,0x3068,0x306f,0x306f,0x3072,0x3072,0x3075,0x3075,0x3078,0x3078,0x3046,0x309d,0x1114,0x1115,0x11c7,0x11c8,0x11cc,0x11ce,0x691,0x6a9,0x6a9,0x6af,0x6af,0x6b3,0x6b3,0x6b1,0x6b1,0x6ba,0x6ba,0x6bb,0x6bb,0x684,0x684,0x683,0x683,0x686,0x686,0x687,0x687,0x68d,0x68d,0x68c,0x68c,0x68e,0x68e,0x688,0x688,0x698,0x698,0x6c9,0x6be,0x6be,0x6ad,0x6ad,0x6cb,0x6cb,0x6c5,0x6c5,0x5f2,0x5e2,0x5d3,0x5d4,0x5db,0x5dd,0x5e8,0x5ea,0x5e9,0x5e9,0x5d1,0x5d2,0x5d5,0x5d6,0x5d8,0x5d9,0xa76f,0x67e,0x680,0x680,0x67a,0x67a,0x67f,0x67f,0x679,0x679,0x6a4,0x6a4,0x6a6,0x6a6,0x5da,0x5de,0x5e0,0x5e1,0x5e3,0x5e4,0x5e6,0x5e7,0x671,0x671,0x67b,0x67b,0x2212,0x2010,0x2277,0x227a,0x227b,0x2282,0x2283,0x2286,0x2287,0x22a2,0x22a8,0x22a9,0x22ab,0x227c,0x227d,0x2291,0x2292,0x22b2,0x22b3,0x22b4,0x22b5,0x2add,0x2d61,0x3012,0x304d,0x304f,0x3051,0x3053,0x3055,0x3057,0x3059,0x305b,0x305d,0x305f,0x3061,0x3064,0x3066,0x127,0x190,0x2194,0x21d0,0x21d4,0x21d2,0x2203,0x2208,0x220b,0x2223,0x2225,0x223c,0x2243,0x2245,0x2248,0x2261,0x224d,0x2264,0x2265,0x2272,0x2273,0x2276,0x2211};
```

Comment:
I'd prefer to revert it, otherwise I don't understand the actual change here.

Reply:
I removed redundant namespace